### DISCUSSION TITLE
command for custom  model creation

### DISCUSSION THREAD

**Discussion by ibalaji777** [adonisjs/core#2997] [Help]
how to create custom model in adonis js ?is there any command for model creation in adonis?

i am using couchdb  in my project ,i don't want lucid in my project .how to create empty model without lucid?

```
import { DateTime } from 'luxon'
import Hash from '@ioc:Adonis/Core/Hash'
import {
  column,
  beforeSave,
  BaseModel,
} from '@ioc:Adonis/Lucid/Orm'

export default class authCloud extends BaseModel {
  @column({ isPrimary: true })
  public id: number

  @column()
  public email: string

  @column({ serializeAs: null })
  public password: string

  @column()
  public rememberMeToken?: string

  @column.dateTime({ autoCreate: true })
  public createdAt: DateTime

  @column.dateTime({ autoCreate: true, autoUpdate: true })
  public updatedAt: DateTime

  @beforeSave()
  public static async hashPassword (authCloud: authCloud) {
    if (authCloud.$dirty.password) {
      authCloud.password = await Hash.make(authCloud.password)
    }
  }
}

```

```
i found below code in third party website related to adonis v4
const Model = use('Model');

class Couchdb extends Model {



}

module.exports = Couchdb ;
```
ReferenceError: use is not defined
const Model = use('Model');

**Comment by RomainLanz** [comment]
Hey @ibalaji777! 👋 

What are you calling Modal?

If it is a popup on the client, this is not related to AdonisJS and you can use any UI Kit your like (like Boostrap, Bulma, etc.) and use those.

**Reply by ibalaji777** [reply]
> Hey @ibalaji777! 👋
> 
> What are you calling Modal?
> 
> If it is a popup on the client, this is not related to AdonisJS and you can use any UI Kit your like (like Boostrap, Bulma, etc.) and use those.

sorry  my typo mistake model not modal
i need to use couchdb  like userModel,adminModel instead lucid

**Comment by McSneaky** [comment]
Since you don't use most of Lucid anyway, I'd skip using Lucid models too

Or then make your own `BaseModel` and call it something like `CouchBaseModel` and write your own models to extend that, rahter than default `BaseModel`
